### ISSUE TITLE
The login dialog sometimes doesn't render on a Nexus 5X on the Google emulator

### ISSUE THREAD

**Issue by puf** [firebase/FirebaseUI-Android#38]
(Related: https://github.com/firebase/FirebaseUI-Android/issues/34)
On the Android emulator with a Nexus 5X image I always get a blank popup, even with just the password provider. Interestingly enough the controls seem to be in there, as I can click and tab. So it's likely a rendering issue.
![screen shot 2015-12-07 at 3 08 59 pm](https://cloud.githubusercontent.com/assets/913631/11642810/e993b4a0-9cf4-11e5-973a-ef019b7588fa.png)

**Comment by jacklt** [comment]
Same issue also on real device: Nexus 6P with Android 6.0.1 (Build MMB29M)

**Comment by abehaskins** [comment]
I'm unable to reproduce this on either device emulator.

@jacklt are you seeing this in the sample app or using the FirebaseLoginDialog in your own application?

**Comment by jacklt** [comment]
@abehaskins I'm using FirebaseLoginDialog in my own application.
I extended FirebaseLoginBaseActivity and call showFirebaseLoginPrompt()

It works on Moto G (3rd ed) with Android 5.1.1
It does not work on Nexus 6P with Android 6.0.1

**Comment by jacklt** [comment]
I tried with the sample app and it works! (on Nexus 6P)

I update the gradle file with this configuration:
```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 23
    buildToolsVersion '23.0.2'

    defaultConfig {
        applicationId "com.firebase.uidemo"
        minSdkVersion 16
        targetSdkVersion 23
        versionCode 1
        versionName "1.0"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
    packagingOptions {
        exclude 'META-INF/LICENSE'
        exclude 'META-INF/NOTICE'
        exclude 'META-INF/NOTICE.txt'
    }

}

dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    compile 'com.android.support:appcompat-v7:23.1.1'
    // compile 'com.google.android.gms:play-services-auth:8.3.0'
    compile 'com.firebase:firebase-client-android:2.3.1'
    compile 'com.android.support:recyclerview-v7:23.1.1'
    //compile 'com.facebook.android:facebook-android-sdk:4.6.0'
    compile project(':library')
}```

and it stops working!

I think it's something related to targetSdkVersion 23

**Comment by abehaskins** [comment]
@jacklt Yep, we're discovering the same thing - in the meantime it appears to function properly with SDK 22 so building with that will fix your issue until we can resolve it.

**Comment by abehaskins** [comment]
Fix committed - coming in 3.1!
